### PR TITLE
Update 22-to-30.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/migration/22-to-30.md [hunk 1]
```diff
@@ -135,15 +140,25 @@ For more information, see the following GitHub resources:
...
 
-For apps that target earlier versions of ASP.NET Core, Kestrel adds HTTP/1.1 chunked trailer headers into the request headers collection when the request body is read to the end. This causes some concerns about ambiguity between headers and trailers, so the trailers have been moved to a new collection (`RequestTrailerExtensions`) in 3.0.
+For apps that target earlier versions of ASP.NET Core:
+
+* Kestrel adds HTTP/1.1 chunked trailer headers into the request headers collection.
+* Trailers are available after the request body is read to the end.
+
+This causes some concerns about ambiguity between headers and trailers, so the trailers have been moved to a new collection (`RequestTrailerExtensions`) in 3.0.
+
+HTTP/2 request trailers:
```
Comment (guardrex) [inline aspnetcore/migration/22-to-30.md]:
```suggestion
HTTP/2 request trailers are:
```

File: aspnetcore/migration/22-to-30.md [hunk 2]
```diff
@@ -135,15 +140,25 @@ For more information, see the following GitHub resources:
...
+
+* Are  not available in ASP.NET Core 2.2.
```
Comment (guardrex) [inline aspnetcore/migration/22-to-30.md]:
```suggestion
* Not available in ASP.NET Core 2.2.
```

File: aspnetcore/migration/22-to-30.md [hunk 3]
```diff
@@ -135,15 +140,25 @@ For more information, see the following GitHub resources:
...
+* Are available in 3.0 as `RequestTrailerExtensions`.
```
Comment (guardrex) [inline aspnetcore/migration/22-to-30.md]:
```suggestion
* Available in 3.0 as `RequestTrailerExtensions`.
```

**Comment by Rick-Anderson** [comment]
@guardrex  Awesome work as always.

**Comment by guardrex** [comment]
I liked ur updates ... I always write paragraphs like I'm working on a SciFi novel! :smile: (... and hopefully soon *I will!* :rocket: 👽)

**Comment by guardrex** [comment]
Did it go in ok? ... some warnings popped up ...


Warning | User | invalid-moniker-range | aspnetcore/security/cors.md | No "::: moniker-end" found for ">= aspnetcore-2.2", MonikerRange does not end explictly. |   | 09/19/2019 00:36:52
-- | -- | -- | -- | -- | -- | --
Warning | User | invalid-moniker-range | aspnetcore/security/cors.md | No "::: moniker-end" found for ">= aspnetcore-3.0", MonikerRange does not end explictly. |   | 09/19/2019 00:36:52
Warning | User | invalid-moniker-range | aspnetcore/security/cors.md | No "::: moniker-end" found for "<= aspnetcore-2.2", MonikerRange does not end explictly. |   | 09/19/2019 00:36:52

**Comment by guardrex** [comment]
Oh :smile: those are from something else (or a point-in-time thing). No worries.

**Comment by Rick-Anderson** [comment]
I think it's because I S&M before it validated. I figured I check validation on the other end.

**Comment by guardrex** [comment]
Do u want me to go look at that topic and make sure it's ok?

**Pull request merged**